pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/ConnectionTimeouts.java

```diff
@@ -25,18 +25,26 @@ public class ConnectionTimeouts {
     private final int _readTimeoutMs;
     private final int _connectTimeoutMs;
     private final int _handshakeTimeoutMs;
+    private final Integer _requestTimeoutMs;
```

Comment:
Would like to keep this optional, not change the default behavior of the existing clients which are using asyncHttpClient's default request timeout (which is 60 seconds i believe) when they upgrade to new version.

Reply:
I see. In that case, you can just drop in the default request timeout in the constructor, instead of using null.
```
  # Replace
        return create(readTimeoutMs, connectTimeoutMs, handshakeTimeoutMs, null);
 # with 
        return create(readTimeoutMs, connectTimeoutMs, handshakeTimeoutMs, 60 * 1000);
```

This will keep it backwards compatible and flexible. I would like to avoid unnecessary boxing/unboxing due to Integer object type. Feels unnecessary.

Reply:
The intent was to not to set any default value here unless the user has spcified it and continue using current behavior (AsyncHttpClient's default timeout). Was deciding between Integer object with null vs Optional, instead of small int with 0/-1 to indicate the user not setting the request timeout property. Didn't see Optional being used in the code, so used null checks for both threadpool name and request timeout. 

I think we can set 60 secs default, which may be okay. Was just playing it safe.  If you guys are fine with it, i can set 60s default (which is what we use for read time out).